middleware/cache/cache_test.go

```diff
@@ -291,15 +291,40 @@ func Test_CustomExpiration(t *testing.T) {
 	}}))
 
 	app.Get("/", func(c *fiber.Ctx) error {
-		c.Response().Header.Add("Cache-Time", "6000")
-		return c.SendString("hi")
+		c.Response().Header.Add("Cache-Time", "2")
+		now := fmt.Sprintf("%d", time.Now().UnixNano())
+		return c.SendString(now)
 	})
 
-	req := httptest.NewRequest("GET", "/", nil)
-	_, err := app.Test(req)
+	resp, err := app.Test(httptest.NewRequest("GET", "/", nil))
 	utils.AssertEqual(t, nil, err)
 	utils.AssertEqual(t, true, called)
-	utils.AssertEqual(t, 6000, newCacheTime)
+	utils.AssertEqual(t, 2, newCacheTime)
+
+	// Sleep until the cache is expired
+	time.Sleep(3 * time.Second)
```

Comment:
please reduce the time for this, otherwise the test procedure will be greatly extended

you can also wait nanoseconds, micro or milliseconds, whichever is stable

if necessary i would decide for a few milliseconds, because the process also takes a bit of time

Reply:
I tried setting `newCacheTime` to `999` and it failed.

Reply:
Also, I don't know if it's related or not. But, I played around with `Test_CacheExpired` and when `Expiration` is set to less than `1 Second`, somehow the expiration time is `1 Minute`

The below test fails.
```
t.Parallel()

	app := fiber.New()
	app.Use(New(Config{Expiration: 500 * time.Millisecond}))

	app.Get("/", func(c *fiber.Ctx) error {
		return c.SendString(fmt.Sprintf("%d", time.Now().UnixNano()))
	})

	resp, err := app.Test(httptest.NewRequest("GET", "/", nil))
	utils.AssertEqual(t, nil, err)
	body, err := ioutil.ReadAll(resp.Body)
	utils.AssertEqual(t, nil, err)

	// Sleep until the cache is expired
	time.Sleep(1000 * time.Millisecond)

	respCached, err := app.Test(httptest.NewRequest("GET", "/", nil))
	utils.AssertEqual(t, nil, err)
	bodyCached, err := ioutil.ReadAll(respCached.Body)
	utils.AssertEqual(t, nil, err)

	if bytes.Equal(body, bodyCached) {
		t.Errorf("Cache should have expired: %s, %s", body, bodyCached)
	}
```